sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -95,6 +95,7 @@
 #include "lardataobj/RecoBase/Vertex.h"
 #include "lardataobj/RecoBase/Shower.h"
 #include "lardataobj/RecoBase/MCSFitResult.h"
+#include "lardataobj/RecoBase/Cluster.h"
```

Comment:
I'm only leaving this as comments for now as I have more pieces to look at and so don't want to leave an actual "review" result yet, but I'll put a couple thoughts here.

I guess the inclusion of `FillReco` prevents this from being an actual problem, but I'm trying to recall if our style guidelines suggest that you should actually put
```
#include "lardataobj/AnalysisBase/MVAOutput.h"
```
here as an include too since `FeatureVector` is directly used in the CAFMaker module.

Reply:
sure, I'll add this include!

Reply:
Thanks for adding!